weed/server/volume_grpc_client_to_master.go

```diff
@@ -116,6 +119,12 @@ func (vs *VolumeServer) doHeartbeat(masterAddress pb.ServerAddress, grpcDialOpti
 				doneChan <- err
 				return
 			}
+			if in.HasDuplicatedDirectory {
+				glog.Error("Shut Down Volume Server due to duplicated volume directory")
+				glog.V(0).Infof("send SIGINT to Volume Server")
+				p, _ := os.FindProcess(vs.pid)
```

Comment:
Why not just exit here?

Reply:
At first I also wanted to exit here, it's much easier. But at this time, grpc server and http server have been started. 
To close them safely, I send  SIGINT to volume server just like the normal exit.

Reply:
Here is a demo log:
```
0517 10:41:28 30304 volume_grpc_client_to_master.go:110] Heartbeat to: 10.10.10.61:26000
E0517 10:41:28 30304 volume_grpc_client_to_master.go:123] Shut Down Volume Server due to duplicated volume directory
I0517 10:41:28 30304 volume_grpc_client_to_master.go:124] send SIGINT to Volume Server
volume server has be killed
I0517 10:41:28 30304 volume_server.go:127] Stopping volume server...
I0517 10:41:28 30304 volume_grpc_client_to_master.go:72] heartbeat error: rpc error: code = Unknown desc = volume: Duplicated volume directories were loaded
I0517 10:41:28 30304 volume.go:275] stop send heartbeat and wait 10 seconds until shutdown ...
I0517 10:41:38 30304 volume.go:299] graceful stop cluster http server ...
I0517 10:41:38 30304 volume.go:304] graceful stop gRPC ...
I0517 10:41:38 30304 volume_server.go:132] Shutting down volume server...
I0517 10:41:39 30304 volume_server.go:134] Shut down successfully!
```

Reply:
They are not expected to serve requests if having wrong configurations.

Reply:
Got it, I'll remove this logic